<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Graphs</class>
 <widget class="QWidget" name="Graphs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>700</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Таблицы и графики</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="GraphsWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="ValuesTab">
      <attribute name="title">
       <string>Таблицы разбиения</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="ValuesEvenLabel">
         <property name="text">
          <string>Равномерное разбиение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="ValuesEvenTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <row>
          <property name="text">
           <string>x</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </row>
         <row>
          <property name="text">
           <string>f(x)</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </row>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ValuesChebyshevLabel">
         <property name="text">
          <string>Чебышевское разбиение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="ValuesChebyshevTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>2</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <row>
          <property name="text">
           <string>x</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </row>
         <row>
          <property name="text">
           <string>f(x)</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </row>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="DivisionsTab">
      <attribute name="title">
       <string>Таблицы разделенных разностей</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="DivisionsEvenLabel">
         <property name="text">
          <string>Равномерное разбиение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="DivisionsEvenTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="DivisionsChebyshevLabel">
         <property name="text">
          <string>Чебышевское разбиение</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="DivisionsChebyshevTable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="FunctionGraphsTab">
      <attribute name="title">
       <string>График функции + полинома</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="1">
        <widget class="QChartView" name="FunctionGraphsChebyshevChart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="FunctionGraphsChebyshevLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Чебышевское разбиение</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="FunctionGraphsEvenLabel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Равномерное разбиение</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QChartView" name="FunctionGraphsEvenChart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ErrorsTab">
      <attribute name="title">
       <string>Таблица + график погрешности</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTableWidget" name="ErrorsTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="rowCount">
          <number>1</number>
         </property>
         <property name="columnCount">
          <number>3</number>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <row/>
         <column>
          <property name="text">
           <string>степень
интерполяционного полинома</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>погрешность интерполяции
при равномерном разбиении</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>погрешность интерполяции
при Чебышевском разбиении</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QChartView" name="ErrorGraphsChart">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QGraphicsView</extends>
   <header>QtCharts</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
